src/song-card/song-card.tsx

```diff
@@ -125,6 +126,16 @@ export function SongCard(props: Props) {
   }
 
   const iconCallbacks = useIconCallbacksForChart((chart as DrawnChart).id);
+  const handleCopy = useCallback(() => {
+    if (!diffAbbr) {
+      return;
+    }
+    copyTextToClipboard(
+      `${name} [${diffAbbr.toUpperCase()}]`,
+      "Copied name & difficulty",
+    );
+  }, [name, diffAbbr]);
```

Comment:
by defining it as a local callback here rather than in the store it's now easy to have it work anywhere a card is used (i.e. in the eligible charts list)